client/js/components/input.js

```diff
@@ -5,7 +5,8 @@ import cx from 'classnames'
 const Input = ({ classes, input, type, placeholder }) => {
 	const inputClasses = cx(
 		classes.textInput,
-		input.name === 'prcName' && classes.largeFontSize,
+		(input.name === 'prcName' || input.name === 'nameOfCHC') &&
```

Comment:
@bhavyaab To keep the naming consistent (and also to use true camel case naming), what do you think of calling this `chcName`?